xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -1022,7 +1022,7 @@ void CPVRClients::StartChannelScan(void)
   long perfCnt = XbmcThreads::SystemClockMillis();
 
   /* stop the supervisor thread */
-  g_PVRManager.StopUpdateThreads();
+  g_PVRManager.Stop();
```

Comment:
to prevent things like the add-on instance being destroyed while the scan is active iirc. if it uses smart ptrs now, then this is probably no longer needed.